Sync documentation

miguelangel-dev opened the pull request:
## Details
Update the nef documentation to keep in sync with the next release 0.6+ Also, I have re-distribute the whole sections and I have added another more to link to api-docs.

## Sections
- Quick start
- Command-Line
- Library
- First steps

![Captura de pantalla 2020-03-20 a las 11 52 09](https://user-images.githubusercontent.com/25568562/77157258-4bab0700-6aa1-11ea-8c58-838ea0c8cd99.png)

miguelangel-dev committed:
update documentation to next nef features

miguelangel-dev committed:
update the whole examples to keep it sync with the next nef release 0.6+

miguelangel-dev committed:
complete apis information and add reference to api-docs

miguelangel-dev committed:
update first step name section

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Compiling a nef Playground.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,31 @@
+ 
+ Where `<nef playground>` is the path to `nef Playground` where your playgrounds are located. Also, you can use the following option with this command:
+ 
+ - `--use-cache`: Use cached dependencies if it is possible, in another case, it will download them. Example:
```
```suggestion
 - `--use-cache`: Use cached dependencies if it is possible; otherwise, it will download them. Example:
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,42 @@
+/*:
+ ## 📲 Creating a Playground Book for iPad
+ 
+ Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. In the latest updates, Swift Playgrounds 3.x has added a new feature: [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it lets you include swift code and make it available across multiple ...
```
This may get old and we won't remember to update it, so a better, future-proof way of wording it:
```suggestion
 Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. Starting from Swift Playgrounds 3.x, the app has added a new feature: [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it lets you include Swift code and make it available across multiple chapters, like modules.
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,42 @@
+ 
+ Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. In the latest updates, Swift Playgrounds 3.x has added a new feature: [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it lets you include swift code and make it available across multiple ...
+ 
+ nef takes advantage of these new possibilities and advancements in Swift Package Manager to build a Playground Book with external dependencies from a Swift Package specification.
```
```suggestion
 nef takes advantage of these new possibilities, together with advancements in Swift Package Manager, to build a Playground Book with external dependencies from a Swift Package specification.
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,42 @@
+ dependencies: [
+ .package(url: "https://github.com/bow-swift/bow.git", from: "0.7.0"),
+ ]
+ )
```
```suggestion
 let package = Package(
   name: "BowProject",
   dependencies: [
     .package(url: "https://github.com/bow-swift/bow.git", from: "0.7.0"),
   ]
 )
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift:
````diff
@@ -0,0 +1,42 @@
+ )
+ ```
+ 
+ you can run the following command:
````
```suggestion
 You can run the following command:
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift:
````diff
@@ -0,0 +1,42 @@
+ ➜ nef ipad --name PlaygroundName --package Package.swift --output ~/Desktop
+ ```
+ 
+ It will create a Playground Book (`PlaygroundName`) with support for the external dependencies and save it in `~/Desktop`
````
```suggestion
 It will create a Playground Book (named `PlaygroundName`) with support for the external dependencies, and save it in `~/Desktop`
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Exporting Carbon code snippets.xcplaygroundpage/Contents.swift:
```diff
@@ -12,24 +12,24 @@ Nef.Playground.needsIndefiniteExecution(false)
- Xcode Playgrounds are a great place where prototyping and trying new concepts. Oftentimes we want to share some Swift snippets; for this, `Carbon` is a cool tool and `nef` lovely integrated it. You can take your Xcode Playground, write several pieces of code and keep it verified. Later you can export all your code snippets, with the next command:
-
+ 
+ Xcode Playgrounds are a great place where prototyping and trying new concepts. Oftentimes we want to share some Swift snippets; for this, `Carbon` is a cool tool and `nef` lovely integrated it. You can take your nef Playground, write several pieces of code and keep it verified. Later you can export all your code snippets, with the next command:
```
```suggestion
Xcode Playgrounds are a great tool for prototyping and trying new concepts. Oftentimes we want to share some Swift snippets; for this, `Carbon` is a cool tool, and `nef` integrates seamlessly with it. You can take your nef Playground, write several pieces of code and keep it verified. Later you can export all your code snippets, with the following command:
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Exporting Carbon code snippets.xcplaygroundpage/Contents.swift:
```diff
@@ -124,3 +124,7 @@ Nef.Playground.needsIndefiniteExecution(false)
  */
+
+/*:
+ You can find an example of use in the section [How to export my snippets?](http://127.0.0.1:4000/docs/first-steps/how-to-export-my-snippets-/)
```
```suggestion
 You can find an example of use in the section [How to export my snippets?](/docs/first-steps/how-to-export-my-snippets-/)
```

miguelangel-dev replied:
nice catch!

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Generating a Markdown document.xcplaygroundpage/Contents.swift:
```diff
@@ -13,27 +13,27 @@ Nef.Playground.needsIndefiniteExecution(false)
-/•:
-‌‌ ### This is a heading 1
+ /•:
+ ‌‌ ### This is a heading 1
```
Heading 1 is only one `#`

```suggestion
 ‌‌ # This is a heading 1
```

truizlop commented on Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Generating a Markdown document.xcplaygroundpage/Contents.swift:
```diff
@@ -13,27 +13,27 @@ Nef.Playground.needsIndefiniteExecution(false)
-‌‌ ### This is a heading 1
+ /•:
+ ‌‌ ### This is a heading 1
  This is regular text. *This is bold text*. [This is a link](http://bow-swift.io).
```
Bold text is 2 asterisks.

```suggestion
 This is regular text. **This is bold text**. [This is a link](http://bow-swift.io).
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to export my snippets?.xcplaygroundpage/Contents.swift:
```diff
@@ -45,18 +39,18 @@ In this tutorial, we assume you do not have any project, and we will follow up f
  ### 3. Generate code snippets
 
- Open a terminal and go to your `nef` project, in our case to `TutorialCarbon` and type
+ Using your terminal we will export the whole code snippets using `nef` and your `nef Playground`, in our case to `TutorialCarbon`.
```
```suggestion
 Using your terminal we will export all code snippets using `nef` and your `nef Playground`, in our case from `TutorialCarbon`.
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to export my snippets?.xcplaygroundpage/Contents.swift:
```diff
@@ -66,5 +60,5 @@ In this tutorial, we assume you do not have any project, and we will follow up f
  &nbsp;
  ### 4. Customize your snippets!
- Go to [Carbon Integration](/docs/integration/carbon/) section for reading more about these and another parameters you can use :)
+ Go back to [Carbon Integration](/docs/command-line/exporting-carbon-code-snippets/) for reading more about these and another parameters you can use :)
```
```suggestion
 Go back to [Carbon Integration](/docs/command-line/exporting-carbon-code-snippets/) to read more about these and other parameters you can use :)
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,59 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: How to use carbon api?
```
The file name should be changed accordingly, but I cannot do it from here.

```suggestion
 title: How to use Carbon API?
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,59 @@
+
+/*:
+ 
+ ## How to use carbon library?
```
```suggestion
 ## How to use Carbon API?
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,59 @@
+ In this section, we will see how we can use the `Carbon API`.
+ 
+ Firstly, you need to import nef library in your project.
+ > If you need to know how to add nef dependency to your macOS project, read [How to use nef library?](/docs/library/how-to-use-nef-library-/) section.
```
```suggestion
 > If you need to know how to add nef dependency to your macOS project, read [How to use nef as a library?](/docs/library/how-to-use-nef-library-/) section.
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
````diff
@@ -0,0 +1,59 @@
+ import nef
+ ```
+ 
+ Now, you can create a carbon style to apply to our code, for the generated snippet.
````
```suggestion
 Now, you can create a Carbon style to apply to our code, for the generated snippet.
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
````diff
@@ -0,0 +1,59 @@
+ let io: EnvIO<nef.Console, Error, Data> = nef.Carbon.render(code: code, style: style)
+ ```
+
+ You will receive an [**EnvIO**](https://bow-swift.io/docs/effects/suspending-side-effects/); it lets you suspend the execution of the sides effects, in the example to create the Carbon image. That way, we could combine and compose with other operations. It is powerful!
````
```suggestion
 You will receive an [**EnvIO**](https://bow-swift.io/docs/effects/suspending-side-effects/), which lets you suspend the execution of the side effects; in the example, to create the Carbon image. That way, we could combine and compose it with other operations.
```

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift:
````diff
@@ -0,0 +1,59 @@
+ }^.mapLeft { error in
+     // TODO: handle error, in case something goes wrong
+ }
+ ```
````
This example is misleading, as invoking the `unsafe` operation should be the last thing to do. We don't want to promote incorrect usage of these constructions. A different example should be added here.

truizlop commented on Documentation.app/Contents/MacOS/First steps.playground/contents.xcplayground:
```diff
@@ -1,7 +1,7 @@
     <pages>
-        <page name='Jekyll'/>
-        <page name='Carbon'/>
+        <page name='How to use carbon api?'/>
```
This page should be renamed.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+/*:
+ ## APIs
+
+ `nef` library provides you the whole features into your macOS project. It enables you to build new tooling on top of nef.
```
```suggestion
 `nef` provides all its features as a library that you can consume in your macOS project. It enables you to build new tooling on top of nef.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ `nef` library provides you the whole features into your macOS project. It enables you to build new tooling on top of nef.
+ 
+ It gives you an easy access to render Markdown files, Jekyll content and Carbon images in a functional way. Also you can compile or clean your nef Playrounds, even make an Xcode Playground/Playground Book compatible with 3rd-party libraries.
```
```suggestion
 It gives you easy access to render Markdown files, Jekyll content and Carbon images in a functional way. Also you can compile or clean your nef Playgrounds, even make an Xcode Playground or Playground Book compatible with 3rd-party libraries.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Playground
+
+ Xcode Playgrounds are a friendly tool for prototyping and trying new concepts. However, they have some limitations, if you need to create playgrounds with 3rd-party libraries support or make your Xcode Playground compatibles with the rest of the options.
```
```suggestion
 Xcode Playgrounds are a friendly tool for prototyping and trying new concepts. However, they have some limitations, if you need to create playgrounds with 3rd-party libraries support or make your Xcode Playground compatible with the rest of the options.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ Xcode Playgrounds are a friendly tool for prototyping and trying new concepts. However, they have some limitations, if you need to create playgrounds with 3rd-party libraries support or make your Xcode Playground compatibles with the rest of the options.
+ 
+ You can find more information in [Playground API docs](TODO).
```
Add link here.

miguelangel-dev replied:
yep, I am waiting for your PR :)

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+ 
+ ### Compile
+
+ If you need to compile a nef Playground in the same way you will do from Xcode Playground IDE.
```
```suggestion
 You can compile a nef Playground in the same way you do from Xcode Playground IDE.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ If you need to compile a nef Playground in the same way you will do from Xcode Playground IDE.
+ 
+ You can find more information in [Compile API docs](TODO)
```
Add link here.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Clean
+
+ In case you need to reset a nef Playground, for example to share as light as possible (without any extra compilation file).
```
```suggestion
 You can clean a nef Playground, for example to share it as light as possible (without any extra compilation file).
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Markdown
+
+If you need to render the content of a Playground page, or anything else that combine [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
```
What would be the *anything else* that is not a Playground page?

```suggestion
This API lets you render the content of a Playground page, or anything else that contains [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+If you need to render the content of a Playground page, or anything else that combine [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
+
+ You can find more information in [Markdown API docs](TODO).
```
Add link here.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Jekyll
+
+ If you need to render content that combines markdown, code, and [**nef commands**](https://github.com/bow-swift/nef#-generating-markdown-files-for-jekyll) to create a verified static website or blog.
```
```suggestion
 This API lets you render content that contains markdown, code, and [**nef commands**](https://github.com/bow-swift/nef#-generating-markdown-files-for-jekyll) to create a verified static website or blog.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Carbon
+
+ If you want to share some Swift snippets. Carbon is a cool tool for this, and nef nicely integrates with it. nef lets you render Carbon images given a configuration (including code to render and style)
```
```suggestion
 You can share some Swift snippets as images using this API that integrates with Carbon. nef lets you render Carbon images given a configuration (including code to render and style)
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ ### Playground Book
+
+ Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. This API let you build a Playground Book with external dependencies from a Swift Package specification.
```
```suggestion
 Swift Playgrounds is a useful app that makes possible to write Swift code on an iPad. This API lets you create a Playground Book with external dependencies from a Swift Package specification.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ If you need to render content that combines markdown, code, and [**nef commands**](https://github.com/bow-swift/nef#-generating-markdown-files-for-jekyll) to create a verified static website or blog.
+
+ You can find more information in [Jekyll API docs](TODO).
```
Add link here.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ If you want to share some Swift snippets. Carbon is a cool tool for this, and nef nicely integrates with it. nef lets you render Carbon images given a configuration (including code to render and style)
+
+ You can find more information in [Carbon API docs](TODO).
```
Add link here.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,57 @@
+
+ Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. This API let you build a Playground Book with external dependencies from a Swift Package specification.
+ 
+ You can find more information in [Playground Book API docs](TODO).
```
Add link here.

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,24 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: How to use nef library?
```
The title of the page should be changed accordingly, as well as possible links to this page.

```suggestion
 title: How to use nef as a library?
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,24 @@
+// nef:end
+
+/*:
+ ## How to install nef library?
```
```suggestion
 ## How to use nef as a library?
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,24 @@
+ 
+ As you know, from Xcode 11, you can integrate package dependencies using Swift Package Manager (SPM) to share code between projects or even use third-party libraries. You can read more about it in Apple's article [Adding package dependencies to your app](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app)
+ 
+ Starting on `nef` [version 0.4](https://github.com/bow-swift/nef/releases/tag/0.4.0), we have modularized the core so that you can use nef as in library as in your macOS projects. Taking advantage of the integration of SPM in Xcode, you can easily import `nef` in your project.
```
```suggestion
 Starting on `nef` [version 0.4](https://github.com/bow-swift/nef/releases/tag/0.4.0), we have modularized the core so that you can use nef as a library in your macOS projects. Taking advantage of the integration of SPM in Xcode, you can easily import `nef` in your project.
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift:
```diff
@@ -0,0 +1,24 @@
+ 
+ ![Add nef library to Xcode as a dependency](/assets/nef-xcode-library.png)
+ 
+ Once Xcode resolves the dependencies, you can import wherever you want, as a native framework:
```
```suggestion
 Once Xcode resolves the dependencies, you can import nef wherever you need, as a native framework:
```

truizlop commented on Documentation.app/Contents/MacOS/Library.playground/contents.xcplayground:
```diff
@@ -2,6 +2,6 @@
 <playground version='6.0' target-platform='ios' display-mode='raw'>
     <pages>
         <page name='How to use nef library?'/>
```
The title of this page needs to be changed.

truizlop commented on Documentation.app/Contents/MacOS/Quick start.playground/Pages/Resources.xcplaygroundpage/Contents.swift:
```diff
@@ -5,24 +5,24 @@
- # Resources
+ ## Resources
 
  In this section you can find a list of projects, libraries, presentations or blog posts that use Bow:
```
```suggestion
 In this section you can find a list of projects, libraries, presentations or blog posts that use nef:
```

miguelangel-dev committed:
Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Compiling a nef Playground.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Quick start.playground/Pages/Resources.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Exporting Carbon code snippets.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Generating a Markdown document.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/First steps.playground/Pages/How to export my snippets?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Generating a Markdown document.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/First steps.playground/Pages/How to export my snippets?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

comments in PR

minor changes

miguelangel-dev committed:
update assets for microsite

miguelangel-dev committed:
update api-docs references in documentation

minor changes

miguelangel-dev committed:
update order in the main menu - nef main page

miguelangel-dev committed:
remove unuseful files

truizlop commented on project/UI/Nef/Version.swift:
```diff
@@ -20,11 +21,30 @@ public struct VersionCommand: ParsableCommand {
+            
+            let imageIO: IO<nef.Error, NSImage> = io.provide(console).flatMap(extractImage)^
+            
+            return nef.Version.info()
```
Why is all this code about loading a Carbon image in the Version example?

miguelangel-dev replied:
It should not be there! Ains!
